examples/aes/ta/src/main.rs

```diff
@@ -32,11 +25,9 @@ fn create() -> Result<()> {
 fn open_session(_params: &mut Parameters, sess_ctx: *mut *mut AesCipher) -> Result<()> {
     trace_println!("[+] TA open session");
     let ptr = Box::into_raw(Box::new(AesCipher {
```

Comment:
Free `ptr` when not using it, since your are "forgetting" the ownership of the `AesCipher`.